rs/tests/message_routing/BUILD.bazel

```diff
@@ -26,7 +26,7 @@ system_test_nns(
         "//rs/tests/driver:ic-system-test-driver",
         "@crate_index//:anyhow",
         "@crate_index//:candid",
-        "@crate_index//:ic-cdk",
+        "@crate_index//:ic_cdk_next",
```

Comment:
Same here.

Reply:
`ic-cdk-next` is defined in `bazel/external_crates.bzl` as being pinned to the `4e287ce` commit.